src/object/srv_mod.c

```diff
@@ -94,6 +98,12 @@ obj_tls_fini(const struct dss_thread_local_storage *dtls,
 	     struct dss_module_key *key, void *data)
 {
 	struct obj_tls *tls = data;
+	struct migrate_pool_tls *pool_tls;
+	struct migrate_pool_tls *tmp;
+
+	d_list_for_each_entry_safe(pool_tls, tmp, &tls->ot_pool_list,
+				   mpt_list)
+		migrate_pool_tls_destroy(pool_tls);
```

Comment:
Force destroy without looking refcount? BTW, what if pool destroy is received while migration is in process? Any way to abort migration?

Reply:
oh, before pool destroy, the migration ult will be stopped. see mpt_fini.